### PR TITLE
Sdl ui

### PR DESCRIPTION
**Pull request by lahoising78**

### CODE REVIEW

File: src/gf3d_pipeline.c [hunk 1]
```diff
@@ -167,6 +237,181 @@ void gf3d_pipeline_render_pass_setup(Pipeline *pipe)
...
+
+    fragShaderStageInfo.sType = VK_STRUCTURE_TYPE_PIPELINE_SHADER_STAGE_CREATE_INFO;
+    fragShaderStageInfo.stage = VK_SHADER_STAGE_FRAGMENT_BIT;
+    fragShaderStageInfo.module = pipe->fragModule;
+    fragShaderStageInfo.pName = "main";
+
+    shaderStages[0] = vertShaderStageInfo;
+    shaderStages[1] = fragShaderStageInfo;
+
+    vertexInputInfo.sType = VK_STRUCTURE_TYPE_PIPELINE_VERTEX_INPUT_STATE_CREATE_INFO;
```
Comment (gravypod) [inline src/gf3d_pipeline.c:298]: Are you attaching a vertex buffer as an input?

Comment (lahoising78) [reply src/gf3d_pipeline.c:298]: Yes. I know it's not required, but I'm planning on using that to control where the gui is rendered on screen

Comment (gravypod) [reply src/gf3d_pipeline.c:298]: If you setup the vetextInputInfo and attach that to the pipeline/descriptorset then you need to attach a vertex buffer to the pipeline. I think that's what the spec says?

**Comment by gravypod** [comment]
@lahoising78 is that last commit message accurate? If so, congrats! That's amazing! You should write up a post with some steps you used and send it to DJ/the class.

**Pull request merged**